NullPointerException @ ClirrReportGenerator.java:64

While running `mvn clean  site -X` with the following configuration for clirr

```
<reporting>
	<plugins>
		<plugin>
			<groupId>org.codehaus.mojo</groupId>
			<artifactId>clirr-maven-plugin</artifactId>
			<version>2.7</version>
		</plugin>
	</plugins>
</reporting>
```

I run into the following null pointer exception.

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-site-plugin:3.3:site (default-site) on project simmetrics-core: Execution default-site of goal org.apache.maven.plugins:maven-site-plugin:3.3:site failed. NullPointerException -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.apache.maven.plugins:maven-site-plugin:3.3:site (default-site) on project simmetrics-core: Execution default-site of goal org.apache.maven.plugins:maven-site-plugin:3.3:site failed.
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:224)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:153)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:145)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:116)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:80)
	at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:51)
	at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:128)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:307)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:193)
	at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:106)
	at org.apache.maven.cli.MavenCli.execute(MavenCli.java:862)
	at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:286)
	at org.apache.maven.cli.MavenCli.main(MavenCli.java:197)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:289)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:229)
	at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:415)
	at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:356)
Caused by: org.apache.maven.plugin.PluginExecutionException: Execution default-site of goal org.apache.maven.plugins:maven-site-plugin:3.3:site failed.
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:145)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:208)
	... 20 more
Caused by: java.lang.NullPointerException
	at org.codehaus.mojo.clirr.ClirrReportGenerator$ApiChangeComparator.compare(ClirrReportGenerator.java:64)
	at org.codehaus.mojo.clirr.ClirrReportGenerator$ApiChangeComparator.compare(ClirrReportGenerator.java:57)
	at java.util.TimSort.binarySort(TimSort.java:292)
	at java.util.TimSort.sort(TimSort.java:235)
	at java.util.Arrays.sort(Arrays.java:1438)
	at java.util.List.sort(List.java:478)
	at java.util.Collections.sort(Collections.java:175)
	at org.codehaus.mojo.clirr.ClirrReportGenerator.doApiChangesTable(ClirrReportGenerator.java:600)
	at org.codehaus.mojo.clirr.ClirrReportGenerator.doApiChanges(ClirrReportGenerator.java:453)
	at org.codehaus.mojo.clirr.ClirrReportGenerator.generateReport(ClirrReportGenerator.java:186)
	at org.codehaus.mojo.clirr.ClirrReport.doReport(ClirrReport.java:316)
	at org.codehaus.mojo.clirr.ClirrReport.generate(ClirrReport.java:227)
	at org.codehaus.mojo.clirr.ClirrReport.generate(ClirrReport.java:363)
	at org.apache.maven.plugins.site.ReportDocumentRenderer.renderDocument(ReportDocumentRenderer.java:228)
	at org.apache.maven.doxia.siterenderer.DefaultSiteRenderer.renderModule(DefaultSiteRenderer.java:319)
	at org.apache.maven.doxia.siterenderer.DefaultSiteRenderer.render(DefaultSiteRenderer.java:135)
	at org.apache.maven.plugins.site.SiteMojo.renderLocale(SiteMojo.java:175)
	at org.apache.maven.plugins.site.SiteMojo.execute(SiteMojo.java:138)
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:134)
	... 21 more

```

If I knew what API changed caused it I'd provide a minimal example. Best I can do currently is provide my current project:
 
https://github.com/Simmetrics/simmetrics/tree/feature-v4.0.0

Call with `mvn clean  site -P reports -X`

--- a/src/main/java/org/codehaus/mojo/clirr/ClirrReportGenerator.java
+++ b/src/main/java/org/codehaus/mojo/clirr/ClirrReportGenerator.java
@@ -103,9 +103,11 @@ public class ClirrReportGenerator
           {
               from = apiDiff.getAffectedMethod();
           }
-          else
+          else if (apiDiff.getAffectedField() != null)
           {
               from = apiDiff.getAffectedField();
+          } else {
+              from = "";
           }
           to = difference.getTo();
 

